tests/openedx_tagging/core/tagging/test_api.py

```diff
@@ -205,3 +211,124 @@ def test_tag_object_invalid_tag(self):
                 "course",
             )
         assert "Invalid object tag for taxonomy" in str(exc.exception)
+
+    def test_import_tags_csv(self):
```

Comment:
There's some lines missing from the test coverage for import and export -- could you add tests for them too?

```
openedx_tagging/core/tagging/api.py                                                 145     11     64      9    90%   185, 191, 206, 212->224, 215, 221-222, 224->exit, 287-289, 304, 310, 340->exit
```

Reply:
Sure, thanks!